### DISCUSSION TITLE
Setting up Maven on GitHub

### DISCUSSION THREAD

**Discussion by Toby-Broom** [qudt/qudt-public-repo#1080] [Q&A] [answered]
How do I setup maven on GitHub it seemed like there was a plugin but it's been depreciated.

**Comment by fkleedorfer** [comment]
Either you install maven locally or you push your changes to a branch on your forked repo on github, make a PR to qudt-public-repo and run the formatter on that branch on github. To do the latter on your branch, navigate to 'actions' select the one that says 'format sources and commit to branch' and run it on your branch, then pull the changes back into your local repo

**Reply by fkleedorfer** [reply]
If that was a bit too condensed, see step 10 here: https://github.com/qudt/qudt-public-repo/wiki/GoodGitPractices#Submitting-a-pull-request-for-review

**Reply by steveraysteveray** [reply, marked as answer]
@Toby-Broom, you had mentioned earlier that you did not see any Actions on the website. You may need to re-sync your fork to get them to appear.

**Reply by Toby-Broom** [reply]
Thanks I see it now.  its fine no worries @fkleedorfer
